Update assign-policy-packages.md (#8754)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/assign-policy-packages.md

## Assign a policy package to users

> A policy package in Teams is a collection of predefined policies and policy settings that you can assign to users who have the same or similar roles in your organization. Each policy package is designed around a user role and includes predefined policies and policy settings that support activities typical for that role. Some examples of policy packages are the Education (Teacher) package and Healthcare (Clinical worker) package. To learn more,R64 see [Manage policy packages in Teams](manage-policy-packages.md).

Reviewer: ```suggestion
A policy package in Teams is a collection of predefined policies and policy settings that you can assign to users who have the same or similar roles in your organization. Each policy package is designed around a user role and includes predefined policies and policy settings that support activities typical for that role. Some examples of policy packages are the Education (Teacher) package and Healthcare (Clinical worker) package. To learn more, see [Manage policy packages in Teams](manage-policy-packages.md).
```

> [!IMPORTANT]
> Before you get started, it's important to understand ([precedence rules](assign-policies-users-and-groups.md#precedence-rules)) and ([group assignment ranking](assign-policies-users-and-groups.md#group-assignment-ranking)). Make sure you read and understand the concepts in ([What you need to know about policy assignment to groups](assign-policies-users-and-groups.md#what-you-need-to-know-about-policy-assignment-to-groups)) earlier in this article.

> > A given policy type can be assigned to at most 64 groups, across policy instances for that type.

Reviewer: ```suggestion
>
> A given policy type can be assigned to a maximum of 64 groups across policy instances for that type.
```
